src/test/java/org/codeforamerica/messaging/services/MessageServiceTest.java

```diff
@@ -193,6 +193,51 @@ public void whenMessagesInBatchHaveDifferentStatuses_ThenGetReturnsCorrectStatus
         assertThat(metricsArray).isEqualTo(new int[] {1, 0, 2, 0, 1, 1, 0, 1});
     }
 
+    @Test
+    public void whenRecipientsFileIsMissingHeader_ThenThrowException() {
+        String recipients = """
+                phone, email
+                1234567890, bar@example.org, placeholder
+                8885551212, foo@example.com, placeholder
+                """;
+
+        MessageBatch messageBatch = MessageBatch.builder()
+                .template(template)
+                .recipients(recipients.getBytes())
+                .build();
+        messageBatchRepository.save(messageBatch);
+
+        assertThrows(Exception.class, () -> messageService.scheduleMessageBatch(messageBatch.getId()));
```

Comment:
We could be more specific here, ie, `RecipientsFileMissingHeadersException.class`.